mempool.go

```diff
@@ -500,11 +500,6 @@ func (mp *txMemPool) maybeAddOrphan(tx *btcutil.Tx) error {
 	// orphans.  In the case there is a valid transaction larger than this,
 	// it will ultimtely be rebroadcast after the parent transactions
 	// have been mined or otherwise received.
-	//
-	// Note that the number of orphan transactions in the orphan pool is
-	// also limited, so this equates to a maximum memory used of
-	// maxOrphanTxSize * maxOrphanTransactions (which is 500MB as of the
```

Comment:
Please leave the note here.  Just modify it to say * cfg.MaxOrphanTransactions and `which is 500MB with the default values as of the time this comment was written).